internal/services/v1/experimental.go

```diff
@@ -497,6 +497,81 @@ func (es *experimentalServer) ExperimentalDiffSchema(ctx context.Context, req *v
 	return resp, nil
 }
 
+func (es *experimentalServer) ExperimentalComputablePermissions(ctx context.Context, req *v1.ExperimentalComputablePermissionsRequest) (*v1.ExperimentalComputablePermissionsResponse, error) {
```

Comment:
I'm also thinking there is an opportunity to support providing multiple namespace+relations pairs. If an update comes via Watch API and a transaction changes various relations, it could become very chatty over the wire to check each relation one by one.

Reply:
I didn't think it was necessary, but we could add it as a followup. As for the multiple pairs, the expectation is that the caller will cache this information

Reply:
👍🏻 Let's make that explicit in the docs for now then